### DISCUSSION TITLE
Apply to be a Leopard Mentor: If you know JavaScript, we need your help!

### DISCUSSION THREAD

**Discussion by PullJosh** [leopard-js/leopard-mentors#1] [Announcements]
Hello Scratchers! As I'm sure you know, Leopard is an online JavaScript community for Scratchers who are graduating to more advanced programming.

There are many new Leopard users who don't know JavaScript and who could use a little help. In particular, sometimes the project translator doesn't work 100% correctly and users need help fixing their projects.

To be a Leopard Mentor, you do NOT need to be a complete JavaScript expert. However, you do need to…
- Have some amount of JavaScript experience and be willing to help solve problems
- Be a patient teacher and thoughtful collaborator who can support new users and encourage them on their journey to learn JavaScript

Leopard Mentors are responsible for answering questions on this discussion board as well as reporting bugs and glitches in Leopard so that they can be fixed.

If you become a mentor, your profile picture will be added to the random rotation that appears directly in the project editor. If you don't want this (or if you have specific preferences about how your logo appears), let me know.

<img width="545" alt="image" src="https://github.com/leopard-js/leopard-mentors/assets/5458180/62806a7f-8319-45bf-91c4-f465f27212cf">

---

**If you are interested in becoming a mentor, please leave a comment below.** In your comment, please include:
1. A link to your Scratch account
2. A link to something you've made with JavaScript (it could be a Leopard project or something else)
3. A short (2-3 sentence) description of why you want to be a mentor. (A quote from this description may appear on [the official list of mentors](https://github.com/leopard-js/leopard-mentors/discussions/2).)

If you mentor application is approved, you will be added to the official mentor list.

**Comment by bates64** [comment]
👋

Scratch accounts: [1](https://scratch.mit.edu/users/grannycookies/) [2](https://scratch.mit.edu/users/nanalan/) [3](https://scratch.mit.edu/users/nanaian/) [4](https://scratch.mit.edu/users/cookielawyer/)

The frontend of https://decomp.me/ is all JavaScript - specifically, Next.js / React. [Source code](https://github.com/decompme/decomp.me)

Scratch is awesome. When I was 12, the Advanced Topics taught me how to create custom Scratch 2.0 blocks with JavaScript. Then I realised that _JavaScript is awesome too_ because you can do anything with it - and you don't need Scratch! I'd like to be a mentor because the Scratch-to-JavaScript pipeline is how I got into programming, and now I'm a professional software engineer. Leopard is a great way to jump the gap between the languages, and build cool things with JavaScript whilst still feeling familiar to Scratchers.

**Reply by PullJosh** [reply]
Awesome! Thanks so much! 👍 I've added you to the official list of mentors (#2)

**Comment by towerofnix** [comment]
Scratch accounts: [_nix](https://scratch.mit.edu/users/_nix/) and [liam48D](https://scratch.mit.edu/users/liam48D/)

JavaScript projects: Pretty much our entire GitHub profile, LOL. But we're the main coders behind [hsmusic-wiki](https://github.com/hsmusic/hsmusic-wiki/), which is a lot of interconnected systems we all coded from scratch and have maintained and improved lots over the years. We help with [sb-edit](https://github.com/leopard-js/sb-edit), the code that converts Scratch projects to Leopard, too!

Intro: We loved coding as a community with other Scratchers: we made our own projects and got tips when we asked for help or feedback, and we competed in for-fun contests with our friends... but the most special thing was collaborating on JavaScript projects we created as a community. Since we loved Scratch so much, we kept learning Scratch at the same time as JavaScript. We think people like us who learned from Scratch have a lot of special perspective, and are super creative, so we'd love to help you get more comfortable with JavaScript - this language is a lot of fun, too, and you can do pretty much anything with it!

<details><summary>Short cut intro, LOL</summary>

> We always loved learning and coding as a community with other Scratchers! We think people like us who learned from Scratch have a lot of special perspective, and are super creative, so we'd love to help you get more comfortable with JavaScript - this language is a lot of fun, too, and you can do pretty much anything with it!
</details>

**Reply by PullJosh** [reply]
Fantastic! Thank you so much. You're on the list! #2
